x-pack/legacy/plugins/apm/public/components/shared/charts/TransactionCharts/TransactionLineChart/index.tsx

```diff
@@ -37,11 +38,21 @@ const TransactionLineChart: React.FC<Props> = (props: Props) => {
     yMax = 'max',
     height,
     truncateLegends,
-    stacked = false
+    stacked = false,
+    onHover
   } = props;
 
   const syncedChartsProps = useChartsSync();
 
+  // create callback for syncedChartsProps.onHover and props.onHover
+  const syncedChartsOnHover = syncedChartsProps.onHover;
+  syncedChartsProps.onHover = (...args) => {
+    if (onHover) {
+      onHover();
+    }
+    return syncedChartsOnHover(...args);
+  };
```

Comment:
Will this affect all the consumers of the hook (since we are mutating the object)? And is that a problem (maybe not)?

Reply:
you're right, seems like a risk, i'll remove the object mutation